maven-wrapper-distribution/src/resources/mvnw

```diff
@@ -194,7 +194,9 @@ else
       wrapperUrl="https://repo.maven.apache.org/maven2/org/apache/maven/wrapper/maven-wrapper/@@project.version@@/maven-wrapper-@@project.version@@.jar"
     fi
     while IFS="=" read -r key value; do
```

Comment:
as a side note, don't think it is necesarilly related to this issue, but https://docs.oracle.com/cd/E23095_01/Platform.93/ATGProgGuide/html/s0204propertiesfileformat01.html suggests that `:` is a valid character here too.